tool-plugins/vscode/src/core/extension.ts

```diff
@@ -199,7 +199,8 @@ export class BallerinaExtension {
                 params.affectsConfiguration(OVERRIDE_BALLERINA_HOME) ||
                 params.affectsConfiguration(ALLOW_EXPERIMENTAL) ||
                 params.affectsConfiguration(ENABLE_DEBUG_LOG) ||
-                params.affectsConfiguration(ENABLE_TRACE_LOG)) {
+                params.affectsConfiguration(ENABLE_TRACE_LOG) ||
+                params.affectsConfiguration(ENABLE_STDLIB_DEFINITION)) {
```

Comment:
We need to remove these parameters since we can handle these config changes on-the-fly with the support of the LSP protocol's `didChangeConfiguration` notifications. WDYT?

Reply:
+1, We need to think a proper way for handling all the options such as trace logs and and debug logs as well